torchvision/models/detection/generalized_rcnn.py

```diff
@@ -97,7 +104,7 @@ def forward(self, images, targets=None):
             features = OrderedDict([("0", features)])
         proposals, proposal_losses = self.rpn(images, features, targets)
         detections, detector_losses = self.roi_heads(features, proposals, images.image_sizes, targets)
-        detections = self.transform.postprocess(detections, images.image_sizes, original_image_sizes)
+        detections = self.transform.postprocess(detections, images.image_sizes, original_image_sizes)  # type: ignore[operator]
```

Comment:
Out of curiosity, what's up with that warning?

Reply:
Here is the error.

```
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
torchvision/models/detection/generalized_rcnn.py:106: error: "Tensor" not
callable  [operator]
            detections = self.transform.postprocess(detections, images.ima...
                         ^
Found 1 error in 1 file (checked 143 source files)

Exited with code exit status 1
```
We pass transform as `nn.Module`, the transform `GeneralizedRCNNTransform` has a method called `postprocess` but `GeneralizedRCNN` does not know that. So it things it's a Tensor and not a function, hence the error.

I don't know easy way to fix this, maybe we should type `transform` as `GeneralizedRCNNTransform` ?